src/state/store.ts

```diff
@@ -25,202 +21,213 @@ export interface UIState {
 }
 
 type Actions = {
-    loadInputOptions: () => Promise<void>;
-    loadAllRecipes: () => Promise<void>;
-    selectInput: (inputName: string) => Promise<void>;
-    loadRecipe: (recipeId: string) => Promise<void>;
-    updateRecipeString: (recipeId: string, newString: string) => void;
-    updateRecipeObj: (recipeId: string, updates: Record<string, string | number>) => void;
+    loadAllRecipes: () => void;
+    selectRecipe: (recipeId: string) => Promise<void>;
+    editRecipe: (recipeId: string, updates: Record<string, string | number>) => void;
     restoreRecipeDefault: (recipeId: string) => void;
     getCurrentValue: (path: string) => string | number | undefined;
     getOriginalValue: (path: string) => string | number | undefined;
-    startPacking: (
-        callback: (recipeId: string, configId: string, recipeString: string) => Promise<void>
-    ) => Promise<void>;
+    startPacking: () => Promise<void>;
+    reset: () => void;
 };
 
 export type RecipeStore = RecipeState & UIState & Actions;
 
 const initialState: RecipeState & UIState = {
-    selectedInputName: "",
     selectedRecipeId: "",
-    inputOptions: {},
     recipes: {},
     isLoading: false,
     isPacking: false,
```

Comment:
note of what we talked about: can this be one called `requestStatus` that's an enum of `LOADING`, `PACKING` or `NONE`